tests/e2e/acquire_test.go

```diff
@@ -127,6 +127,34 @@ func TestPoolsToken(t *testing.T) {
 	)
 }
 
+func TestPoolsTypes(t *testing.T) {
+	testenv.Test(t, features.New("resource acquisition by type list").
+		Setup(ofcirSetup("pools-different-types", "pool-0,pool-1")).
+		Assess("allows when type1 available", func(ctx context.Context, t *testing.T, cfg *envconf.Config) context.Context {
+			c := NewOfcirClient(t, cfg, ctx.Value("token").(string))
+			cirInfo := c.TryAcquireCIR("host")
+			assert.Equal(t, "pool-0", cirInfo.Spec.PoolRef.Name)
+			return ctx
+		}).
+		Assess("blocks when type2 not specified", func(ctx context.Context, t *testing.T, cfg *envconf.Config) context.Context {
```

Comment:
This comment is slightly confusing... the size of the pool-0 (type `host`) is 1, so there's no additional host available.
typo: `type2` -> `host2`?